### PR TITLE
11 lts compatibility

### LINKED ISSUE #124: Support for TYPO3 11 ?
Is support for TYPO3 11 in the works?

### PR DESCRIPTION
**Pull request by stigfaerch**
FIXES #124 

But needs this fix on EXT:news:
https://github.com/georgringer/news/pull/1639

I have not tested it thoroughly.

### CODE REVIEW

File: Classes/Controller/NewsController.php [hunk 1]
```diff
@@ -2,6 +2,12 @@
 
 namespace GeorgRinger\Eventnews\Controller;
 
+use GeorgRinger\Eventnews\Domain\Repository\LocationRepository;
+use GeorgRinger\News\Domain\Repository\CategoryRepository;
+use GeorgRinger\News\Domain\Repository\NewsRepository;
+use GeorgRinger\News\Domain\Repository\TagRepository;
+use TYPO3\CMS\Extbase\Configuration\ConfigurationManagerInterface;
```
Comment (kitzberger) [inline Classes/Controller/NewsController.php:9]: @stigfaerch, this contradicts Georgs hint in https://github.com/georgringer/news/blob/master/Documentation/DeveloperManual/ExtendNews/ProxyClassGenerator/Index.rst

> Don't use any use statements as those are currently ignored!

**Review by kitzberger — COMMENTED** [review]
@stigfaerch, I've had a go at your patch. Thanks for the effort!

Seems to be working for `Domain/Model/News.php` but unfortunately not for `NewsController.php`. For the latter I still get an exception in the FE:

> An action "monthAction" does not exist in controller "GeorgRinger\News\Controller\NewsController".

And yes, I've installed the patch for EXT:news you've mentioned: https://patch-diff.githubusercontent.com/raw/georgringer/news/pull/1639.patch

**Comment by stigfaerch** [comment]
@kitzberger I have made some updates. It's working on my system, but of course needs more testing.

**Comment by kitzberger** [comment]
> @kitzberger I have made some updates. It's working on my system, but of course needs more testing.

With EXT:news 9.1 and  TYPO3 10?

**Comment by stigfaerch** [comment]
> > @kitzberger I have made some updates. It's working on my system, but of course needs more testing.
> 
> With EXT:news 9.1 and TYPO3 10?

With TYPO3 11 and EXT:news stigfaerch:fix_class_extension_feature version

**Comment by georgringer** [comment]
thanks for your work. I did some adoptions but didn't go with your original PR

**Pull request closed without merging**